e2e/step_definitions/utility.js

```diff
@@ -40,35 +39,35 @@ defineSupportCode(({Given, Then, When}) => {
 
   // Input a value into a keyed element
   When('I input {string} in the {string}', (value, key) => {
-    return client.useCss().setValue(selectors[key]||key, value);
+    return client.useCss().setValue(selectors[key] || key, value);
   });
 
   // Pause for n seconds
   When('I wait {int} seconds', (seconds) => {
-    return client.pause(seconds*1000);
+    return client.pause(seconds * 1000);
   });
 
   // Check the url
   Then('the url has these values:', (table) => {
-    table.rawTable.forEach(function(row){
+    table.rawTable.forEach(function (row) {
       client.assert.urlParameterEquals(row[0], row[1]);
     });
     return client;
   });
 
   // Check the number of tabs open
   Then('there are {int} tabs open', (count) => {
-    return client.windowHandles(function(tabs) {
+    return client.windowHandles(function (tabs) {
       client.assert.equal(tabs.value.length, count);
     });
   });
 
   // Close a tab by index
   Then('I close tab number {int}', (index) => {
     index--;
-    return client.windowHandles(function(tabs) {
+    return client.windowHandles(function (tabs) {
       client.switchWindow(tabs.value[index]);
-      client.closeWindow(function(){
+      client.closeWindow(function () {
```

Comment:
I like the spacing improvements but this one bugs me,

Reply:
The space between `function` and `()`. The space after is good and all the other rules I saw seem good.

Reply:
It seems a little silly, but in cases where there are params there, it makes things a little easier to read;
```
function (arr, index) {
  // stuff
}
```

But the point of using `standard` is [to avoid bikeshedding](https://standardjs.com/#why-should-i-use-javascript-standard-style) over things like this. The library enforces consistency over personal preferences.

Reply:
👍 @ZachTRice @localjo   I like that space as well

Reply:
One rule that I don't really care for is always having `{` and `}` in if statements. I like to do clean little one-liners like;
```
if (!relevant) return;
```

But I'd rather have consistency and stick to the standard defaults than spending a bunch of time trying to agree on style preferences that at the end of the day aren't super important as long as we're consistent.

Reply:
I'd rather specify standards that work for us. 
`"space-before-function-paren": ["warn", "never"],`
And I think your rule above makes sense too.

These 2 rules are just adding a bit of bloat

Reply:
Actually, that's a bad example, because I don't think that's a `standard` rule. But the point is that we should stick to a standard style rather than personal preferences.

Reply:
Also, sticking to `standard` is good because we're being consistent with [all of these companies](https://standardjs.com/#who-uses-javascript-standard-style), which makes our code feel more familiar for potential contributors.

Reply:
@Benjaki2 what are your thoughts? I lean towards using standard as a guideline and making our own tweaks to that.

Reply:
I think that standard is a good place to start, but we can definitely make our own tweaks to that if we get a majority vote.